src/routes/devcards.ts

```diff
@@ -99,7 +99,12 @@ export default async function (fastify: FastifyInstance): Promise<void> {
         if (format === 'svg') {
           const pngUrl = `${process.env.URL_PREFIX}${req.originalUrl.replace('.svg', '.png')}`;
           const encodedUrl = encode(pngUrl, { useNamedReferences: true });
-          const svgString = `<svg xmlns="http://www.w3.org/2000/svg"><image href="${encodedUrl}" /></svg>`;
+          const svgString = `
+            <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
We should not need the xlink namespace. It's not used anywhere in the svg from what I can see.